src/utils/backend.js

```diff
@@ -111,7 +111,7 @@ export default class Backend {
 
   static getTokenInfo = async () => backendFetch('tokenCache/tokenInfo');
 
-  static getWordRegistry = async (ordering, direction, search) => backendFetch(`tokenCache/wordRegistry?ordering=${ordering}&direction=${direction}&search=${search || ''}`);
+  static getWordRegistry = async (ordering, direction, search) => backendFetch(`tokenCache/wordRegistry?${ordering ? `ordering=${ordering}` : ''}${direction ? `&direction=${direction}` : ''}${search ? `&search=${search}` : ''}`);
```

Comment:
Without `ordering` it will construct the query like `wordRegistry?&direction...`, it still may work well, but I would propose to construct it using URL class. Or finally, generate this API by swagger file then we won't be needed to deal with it.

Reply:
We could start working on the Swagger API generation. The swagger spec should be fairly accurate now (at least regarding the requests). But for now we only need these two minor changes to avoid params sending the value `undefined`. I tried to keep the changes in the one-line fashion but I am happy to adjust them using the URL class.